README.md

```diff
@@ -180,14 +180,10 @@ library archives (`.a`).
 | libreadline  | 6.3.0         | NO       | `libreadline6-dev` | `readline`   | `readline-devel`  | YES      | Input editing  |
 | ldns         | 1.6.17        | NO       | `libldns-dev`      | `ldns`       | `ldns-devel`      | YES      | SSL toolkit    |
 | expat        | 1.1           | NO       | `libexpat1-dev`    | `expat`      | `expat-devel`     | YES      | XML parsing    |
-| GTest        | 1.5           | YES      | `libgtest-dev`^    | `gtest`      | `gtest-devel`     | YES      | Test suite     |
```

Comment:
It _**is**_ already marked as optional.

Reply:
The point is that adding gtest as described will break the build. Only the packaged version works. Hence, why keep instructions in there that do not work?

Reply:
Ah, now I stand corrected, the packaged gtest would cause a build error due to the infamous `-fPIC` issue, so everyone should all be building the vendored gtest with `-fPIC`.

CC @jtgrassie @moneromooo-monero

Reply:
Actually the same seems to apply to other dependencies (i.e. the packaged ones are unusable due to `-fPIC` missing). Am I correct?

Reply:
Depends on your distro. AIUI most are or will be using PIC for static libs.
If libgtest is removed from the README, then the cmake machinery should probably be modified to always use the internal copy to match.